examples/subplots_axes_and_figures/unshare_axis_demo.py

```diff
@@ -0,0 +1,36 @@
+"""
+======================
+Unshare and share axis
+======================
+
+The example shows how to share and unshare axes after they are created.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+t = np.arange(0.01, 5.0, 0.01)
+s1 = np.sin(2 * np.pi * t)
+s2 = np.exp(-t)
+s3 = np.sin(4 * np.pi * t)
+
+ax1 = plt.subplot(311)
+plt.plot(t, s1)
+
+ax2 = plt.subplot(312)
+plt.plot(t, s2)
+
+ax3 = plt.subplot(313)
+plt.plot(t, s3)
+
+ax1.share_x_axes(ax2)
+ax1.share_y_axes(ax2)
+
+# Share both axes.
+ax3.share_axes(ax1)
+plt.xlim(0.01, 5.0)
+
+ax3.unshare_y_axes()
+ax2.unshare_x_axes()
```

Comment:
I didn't try to run this, but its hard to imagine this actually demos anything in a non-interactive session.

Reply:
You do not create share as usual by setting a share parent axes. Sharing is set after the creation of the axes.

Reply:
I think demos should be readily readable by users since they are one of the primary ways we document Matplotlib.   I find this demo is very cryptic.  Why are you un-sharing?  Just to show that these functions exist?  What effect does this have on the example?  The description of the demo should state the usual way of sharing, and this alternate way.

Reply:
hm... no comment.